games-emulation/gw_libretro/gw_libretro-1.6.2_20180125.recipe

```diff
@@ -0,0 +1,54 @@
+SUMMARY="A libretro core for Game & Watch simulators"
+DESCRIPTION="gw-libretro is a libretro core that runs Game & Watch simulators. \
+It runs simulators converted from source code. The converted games are \
+available at libretro's buildbot site. Unpack the zip and read the README.txt \
+for instructions."
+HOMEPAGE="http://bot.libretro.com/assets/cores/Handheld%20Electronic%20Game/"
```

Comment:
```diff
-HOMEPAGE="http://bot.libretro.com/assets/cores/Handheld%20Electronic%20Game/"
+HOMEPAGE="https://bot.libretro.com/assets/cores/Handheld%20Electronic%20Game/"

Reply:
@alaviss: Yep, I noticed that the website also works on https, but maybe they don't want to advertise it too loudly.

@kwyxz: if you think libretro.com would prefer that we stick to http, I think @alaviss would be OK with keeping the http here. (@alaviss, what do you think?)

Reply:
I used http because that is the link they give. I don't think it's a big deal replacing with https but I will ask the team.